share/man/votca-csg.t2t.in

```diff
@@ -1,6 +1,6 @@
 votca-csg
 VOTCA Development Team 
-%%mtime(%d/%m/%Y)
+@VOTCA_DATE@
```

Comment:
Why did you not use a cmake variable for the tools pull request?

Reply:
It felt like an unnecessary dependency for tools as there is only one manpage.

Reply:
On a second thought fixed.